iothub/device/src/ModuleClient.cs

```diff
@@ -44,9 +44,7 @@ internal ModuleClient(InternalClient internalClient, ICertificateValidator certV
             }
 
             if (Logging.IsEnabled)
-            {
```

Comment:
Did you intend to remove these? I haven't seen us choose one way or another in the code so not sure if you're confirming to a code style

Reply:
Yes. I believe that's the agreed upon style for the Logging.IsEnabled block that we discussed. Looking at recent PRs, that's what we've been doing.

Reply:
We decided for the ubiquitous logging only, to reduce how many lines of code it takes.